File: 250228/합과 평균의 차/sub-of-average-and-sum.py
user = input()
arr = user.split()

a = int(arr[0])
b = int(arr[1])
c = int(arr[2])

if 1 <= a <= 100 and 1 <= b <= 100 and 1 <= c <= 100 :
    print(a + b + c)
    print(int((a + b +c) / 3))
    print(int((a + b + c) - ((a + b + c) / 3)))
else :
    print("error")

File: 250721/합과 평균의 차/sub-of-average-and-sum.swift
let input = readLine()!.split(separator: " ").map { Int(String($0))! }
let a = input[0]
let b = input[1]
let c = input[2]


if (1 <= a && a <= 100) && (1 <= b && b <= 100) && (1 <= c && c <= 100) {
    let sum = a + b + c
    let avg = (a + b + c) / 3
    print(sum)
    print(avg)
    print(sum - avg)
}